src/Library/demos/Toggle Button/code.rs

```diff
@@ -0,0 +1,22 @@
+use crate::workbench;
+use gtk::prelude::*;
+use std::collections::HashMap;
+
+pub fn main() {
+    let buttons: HashMap<&str, &str> = HashMap::from([
```

Comment:
The type annotation should not be needed

Reply:
For some reason, it did not work for me without it. Can you help me why? Maybe its just a problem on my side.

Reply:
What is the error message you get?